docs/contributing/css_guidelines.rst

```diff
@@ -49,12 +52,27 @@ Run the linter from the wagtail project root:
 The linter is configured to check your code for adherance to the guidelines
 below, plus a little more.
 
+Formatting code
+---------------
+
 If you want to autofix errors, you can run that command directly with:
 
 .. code-block:: console
 
     $ npm run lint:css -- --fix
 
+Changing the linter configuration
+---------------------------------
+
+To edit ignored files, add an entry to the ``.stylelintrc.yaml`` file under ``ignoreFiles``.
```

Comment:
I know this has been done for JS linting, but I don't think this is worth documenting. We don't want people ignoring files via config (you can do more granularly with disable comments).